### ISSUE TITLE
Generated focus.settings.gradle disappear on clean/rebuild

### ISSUE THREAD

**Issue by ConstOrVar** [dropbox/focus#23]
Plugin works great! But when trigger _clean_ or _rebuild_ (from Android Studio), all build directories will be removed. So generated `focus.settings.gradle` will be deleted and project will be broken (_.focus_ file exists, but points at nowhere now). 
How do you solve that problem in your team?

As workaround I'm replacing path where generated  `focus.settings.gradle` will be saved - select root folder of target project, which placed in focus.

**Comment by rharter** [comment]
Hmm, clean should also be removing your `.focus` file, as it's an `OutputFile` of the focus task. Are you creating it with the `focus` task?

**Comment by ConstOrVar** [comment]
Yes, `.focus` file has been created using `./gradlew module:focus`. And it's still exists after `./gradlew clean` triggered.
If you call rebuild (using Android Studio), first command that is performed, is `clean`. That's why it will be strange if on each rebuild action selected configuration will be lost. In my opinion, that sounds weird.
